packages/extension/src/ui/services/swr.ts

```diff
@@ -104,3 +104,8 @@ export const swrCacheProvider: Cache = {
     }
   },
 }
+
+export const retryWhenRateLimited = (error: any) => {
+  const errorCode = error?.status || error?.errorCode
```

Comment:
Our API error does not have a status or error code. This may not work?

Reply:
Indeed, you are right. 

I have upgraded the SDK version so we can [extract](https://github.com/alephium/extension-wallet/pull/212/commits/ce77bab18a1e86e393f515907974fb55f6c61a2c) the error code. Maybe we should introduce a util function to extract error code in SDK instead?